README.md

```diff
@@ -55,6 +64,51 @@ var globalConfig = {
   
 connect.ChatSession.setGlobalConfig(globalConfig);
 ```
+
+### Message Receipts
+
+Enable message receipts for rendering read/delivered events in chat transcript. Before updating the chat widget, follow these steps to enable message receipts in the admin console:
+
+1. Head to the `Channels` => `Chat Widget` page, then click "edit" on the left hand side:
```

Comment:
Step1 is not needed for custom widgets using github. This is for hosted chat widgets.
For github/custom widgets, the feature is enabled by default so customers only need to follow the example code in this repo to send and display receipts on their UIs